interfaces/IBF-dashboard/src/app/components/disaster-type/disaster-type.component.html

```diff
@@ -1,6 +1,10 @@
-<ion-tab-bar *ngFor="let disasterType of disasterTypes">
-  <ion-tab-button (click)="switchDisasterType(disasterType)">
-    <ion-icon name="warning"></ion-icon>
-    <ion-label>{{ disasterType.label }}</ion-label>
+
+<ion-tab-bar *ngFor="let disasterType of disasterTypes"
+  [ngClass]="{'active': disasterType.label === selectedDisasterType}">
```

Comment:
You can ignore this comment if you choose to. I prefer to use a [consistent class conditional syntax format](https://stackoverflow.com/a/41974490/1753041). [Please take a look at how we've used class conditionals in our other components for reference.](https://github.com/rodekruis/IBF-system/blob/master/interfaces/IBF-dashboard/src/app/components/timeline/timeline.component.html#L18)